bootstrap/Platform_Management/Setup_Custom_Cartridge_Library.groovy

```diff
@@ -0,0 +1,38 @@
+// Constants
+def libraryGitURL = "https://github.com/Accenture/adop-dsl-custom-cartridge-properties.git"
+
+def platformManagementFolderName= "/Platform_Management"
+def platformManagementFolder = folder(platformManagementFolderName) { displayName('Platform Management') }
+
+// Jobs
+def setupCustomCartridge = freeStyleJob(platformManagementFolderName + "/Setup_Custom_Cartridge_Library")
+ 
+// Setup setup_cartridge
+setupCustomCartridge.with{
+    environmentVariables {
+        keepBuildVariables(true)
+        keepSystemVariables(true)
+    }
+    wrappers {
+        preBuildCleanup()
+    }
+    steps {
+        shell('''#!/bin/bash -ex
+
+echo "Extracting Custom_Cartridge library to additonal classpath location: /var/jenkins_home/userContent/job_dsl_additional_classpath/"
+cp -r src/main/groovy/* /var/jenkins_home/userContent/job_dsl_additional_classpath/
```

Comment:
Can this use the JENKINS_HOME environment variable instead?

Reply:
Done!